### ISSUE TITLE
Soft reset fail

### ISSUE THREAD

**Issue by DaniLPGC** [LWooler/Shiny_Pokemon_Emulator_Bots#3]
I am trying to get a shiny Kyurem in Black 2, I am using Desmume. I followed the instructions and the bot seems to work properly but after around 2 or 3 soft resets the bot starts to soft reset at the wrong moment, even before encountering the pokemon

edit: I tried the bot with Heatran, on my Pokemon Diamond game with Cute Charm glitch to see if the bot actually works...
it soft reseted over the shiny...

(the bot screen shows the message: "Warning: 64-bit application should be automated using 64-bit Phyton (you use 32-bit Pyhton)" also, the base colour always shows 240, 240, 240)

**Comment by LWooler** [comment]
Thanks for trying my bot. I tested out Desmume and you are correct it always reads the base color as (240, 240, 240), so I'm sorry it looks like an incompatibility with a library I used and Desmume.

However there is good news, I tested it with RetroArch(x64) and it seems to work with the Desmume core. It may work well with other DS emulators as well.

`I followed the instructions and the bot seems to work properly but after around 2 or 3 soft resets the bot starts to soft reset at the wrong moment, even before encountering the pokemon`

As far as this issue goes, try making your button presses as short as possible or increase the time between button presses. Play around with that and let me know if you still get this issue
